src/xcode/ENA/ENA/Source/Services/Risk/KeyPackageDownload/KeyPackageDownload.swift

```diff
@@ -326,8 +334,8 @@ class KeyPackageDownload: KeyPackageDownloadProtocol {
 			case .daily:
 				downloadedPackagesStore.deleteDayPackage(for: package, country: countryId)
 			case .hourly(let keyDay):
-				// hourly packages for a day are deleted when the day package is stored. See func
-				// DownloadedPackagesSQLLiteStoreV1.set(  country: Country.ID,	day: String, package: SAPDownloadedPackage )
+				// Hourly packages for the last day are deleted when the last day package is stored. See func
+				// DownloadedPackagesSQLLiteStore.set(country: Country.ID, day: String, package: SAPDownloadedPackage)
```

Comment:
Just to be a bit more precise: func DownloadedPackagesSQLLiteStoreV2.set(country: Country.ID, day: String, etag: String?, package: SAPDownloadedPackage) deletes the hour packages for the respective day. This works for all days, not only the last one.

Reply:
Changed the comment to: 
"// Hourly packages for a day are deleted when the day package is stored. See func
// DownloadedPackagesSQLLiteStore.set(country: Country.ID, day: String, package: SAPDownloadedPackage)"

I would like to use the macro DownloadedPackagesSQLLiteStore, because the versioned name would get out of date.